catalog/spatial/geowebcache/geowebcache-app/pom.xml

```diff
@@ -26,7 +26,7 @@
     <name>DDF :: Spatial :: GeoWebCache App</name>
 
     <properties>
-        <gwc-server.version>0.6</gwc-server.version>
+        <gwc-server.version>0.7.0-SNAPSHOT</gwc-server.version>
```

Comment:
sweet! - if you tag me on the GWC PR I can help get that deployed when you're ready.